Controller/controller/ctrlScript.gui_script

```diff
@@ -314,9 +314,9 @@ function on_input(self, action_id, action)
 		end
 	elseif action_id == hash("mouse_left") then
 
-		--for k, button in pairs(self.guiButtons) do
-		--	button.mouseUpdate(action)
-		--end
+		for k, button in pairs(self.guiButtons) do
```

Comment:
Comment this again. It was set for debugging purpose. Otherwise buttons might not work correctly on android devices.